### ISSUE TITLE
Create page dropdown, spacebar response

### ISSUE THREAD

**Issue by rebeccacremona** [harvard-lil/perma#2254]
Why does the folder dropdown open and immediately close when you press spacebar?

It opens, and stays open, when you open it using enter.

**Comment by rebeccacremona** [comment]
This might be a Firefox-only issue.

**Comment by matteocargnelutti** [comment]
Unable to reproduce this issue.
Tested on latest Chrome and Firefox, Mac OS 12.4.
